### ISSUE TITLE
HTML native vscode support language missing with HTML Liquid extension

### ISSUE THREAD

**Issue by diazgilberto** [Microsoft/vscode#29882]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:13:05.928Z)
- OS Version: Darwin x64 16.6.0
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|path-intellisense|christian-kohler|1.4.2|
|jupyter|donjayamanne|1.1.3|
|python|donjayamanne|0.6.5|
|file-icons|file-icons|1.0.4|
|advanced-new-file|patbenatar|0.4.3|
|vscode-icons|robertohuertasm|7.10.1|;

---

Steps to Reproduce:

1. Install Jekyll Snippets extension - 0.8.1
2. The extension will prompt you to install HTM, Liquid, and Markdown as `extensionDependencies`.
3. Try to work in a HTML file that is not related with Jekyll at all.
4. Please notice that the native vscode HTML support language is no longer there.

**Comment by rebornix** [comment]
@diazgilberto do you still run into this issue if you run VSCode without any extension? `code --disable-extensions`

**Comment by diazgilberto** [comment]
No, I don't.
I had to uninstall those extensions in order to see the native vascode HTML support language though.

**Comment by rebornix** [comment]
@diazgilberto if it only happens with extensions, I'm afraid it's caused by one of the extensions, right? Very likely Jekyll Snippets, what do you think

**Comment by diazgilberto** [comment]
I notice that because I was utilizing the keystroke to comment code out and was doing this instead.

    {% comment %}
         This is a block comment in Liquid
    {% endcomment %}

So I went down right to change the syntax and HTML was not there, just HTML Liquid. I went ahead and disabled the extension and Django came by default. Clicked again and HTML was not there. Disabled Django and no HTML was present. When I was trying to uninstall HTML Liquid I was prompted with a window saying that was a Jekyll Snippets dependancy. So I uninstalled both.
